packages/vite-plugin-cloudflare/e2e/basic.test.ts

```diff
@@ -171,7 +176,7 @@ describe("basic e2e tests", () => {
 			describe.skipIf(
 				!process.env.CLOUDFLARE_ACCOUNT_ID || !process.env.CLOUDFLARE_API_TOKEN
 			)("Workers AI", () => {
-				test.skipIf(command === "buildAndPreview")(
+				test.skipIf(isBuildAndPreviewOnWindows(command))(
```

Comment:
Why skip on Windows?

Reply:
Currently this skips if the command is `buildAndPreview` on all OSes, but this is only problematic on Windows machines (the CI is happy on linux and mac), so this reduces to skipped tests to only the problematic ones

do you see an issue with this?

Reply:
Sorry - my mistake